gazebo_msgs/gazebo_msgs_bridge_mapping_rule.yaml

```diff
@@ -0,0 +1,11 @@
+# This file defines mappings between ROS 1 and ROS 2 interfaces.
+# It is used with the ros1_bridge to allow for communcation between ROS 1 and ROS 2.
+
+-
+  ros1_package_name: 'gazebo_msgs'
+  ros1_message_name: 'ODEJointProperties'
+  ros2_package_name: 'gazebo_msgs'
+  ros2_message_name: 'ODEJointProperties'
+  fields_1_to_2:
+   hiStop: hi_stop
+   loStop: lo_stop
```

Comment:
I haven't tried your version yet, but according to the ros1_bridge docs, you have to list all fields, not just the ones with the names changed.

See https://github.com/crdelsey/gazebo_ros_pkgs/commit/6ccbdd3b7e0f398bac46d5b6f8867bc062ea187e

Reply:
Ah you're right, and good call adding the single quotes too. Feel free to open a PR with your branch, otherwise I can add it to this PR, as you prefer.

Reply:
OK. Done. The new PR is #774